doc/source/conf.py

```diff
@@ -25,8 +25,8 @@
 # -- Project information -----------------------------------------------------
 
 project = "xDEM"
-copyright = "2021, Erik Mannerfelt, Romain Hugonnet, Amaury Dehecq and others"
-author = "Erik Mannerfelt, Romain Hugonnet, Amaury Dehecq and others"
+copyright = "2020, GlacioHack"
```

Comment:
I see some packages leaving the original year while others keep it updated. Any ideas on implications of this?

Reply:
No, we should ask advice from CNES/CS on this. Maybe we can adjust when we'll change the license.